core-rust-bridge/src/main/java/com/radixdlt/protocol/ProtocolConfig.java

```diff
@@ -65,22 +65,52 @@
 package com.radixdlt.protocol;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.reflect.TypeToken;
+import com.radixdlt.rev2.NetworkDefinition;
+import com.radixdlt.sbor.NodeSborCodecs;
 import com.radixdlt.sbor.codec.CodecMap;
 import com.radixdlt.sbor.codec.StructCodec;
+import com.radixdlt.sbor.exceptions.SborDecodeException;
+import java.util.Map;
 
 public record ProtocolConfig(
-    String genesisProtocolVersion, ImmutableList<ProtocolUpdate> protocolUpdates) {
+    String genesisProtocolVersion,
+    ImmutableList<ProtocolUpdateTrigger> protocolUpdateTriggers,
+    Map<String, byte[]> rawProtocolUpdateContentOverrides) {
+
+  public static final String GENESIS_PROTOCOL_VERSION_NAME = "babylon-genesis";
+
+  public ProtocolConfig(ImmutableList<ProtocolUpdateTrigger> protocolUpdateTriggers) {
+    this(protocolUpdateTriggers, Map.of());
+  }
+
+  public ProtocolConfig(
+      ImmutableList<ProtocolUpdateTrigger> protocolUpdateTriggers,
+      Map<String, byte[]> rawProtocolUpdateContentOverrides) {
+    this(GENESIS_PROTOCOL_VERSION_NAME, protocolUpdateTriggers, rawProtocolUpdateContentOverrides);
+  }
+
   public static void registerCodec(CodecMap codecMap) {
     codecMap.register(
         ProtocolConfig.class,
         codecs -> StructCodec.fromRecordComponents(ProtocolConfig.class, codecs));
   }
 
+  public static ProtocolConfig sborDecodeWithFallbackForOldVersions(byte[] encoded) {
+    try {
+      return NodeSborCodecs.decode(encoded, NodeSborCodecs.resolveCodec(new TypeToken<>() {}));
+    } catch (SborDecodeException ex) {
```

Comment:
> What if it deserializes [by chance] and just has a different meaning?
It can't - the ProtocolConfig structs have different lengths

> Do we need the backwards compatibility that desperately?
I put this in as a help to the test team because they've already generated about 20 of these, and it would likely be a pain to regenerate this week.

That said, if you feel strongly we can talk to the test team and see if they're comfortable to change; and going forwards, we could make ProtocolConfig a versioned enum.